client/settings/settings.tsx

```diff
@@ -378,6 +391,8 @@ function SettingsSubPageDisplay({
   }
 
```

Comment:
I can't comment on it but you need to remove/change the Electron check above this, this settings page should be available on the web too. If you look at my version in the `gql` branch I have a version of this function that should work + deal with assertUnreachable properly